src/main/scala/io/scalac/tezos/translator/repository/LibraryRepository.scala

```diff
@@ -1,5 +1,19 @@
 package io.scalac.tezos.translator.repository
 
+import io.scalac.tezos.translator.model.LibraryModel
+import io.scalac.tezos.translator.schema.LibraryTable
+import io.scalac.tezos.translator.schema._
+import slick.jdbc.MySQLProfile.api._
+
 class LibraryRepository {
 
+  val acceptedStatus = 1
```

Comment:
maybe companion object is better place for such constants - it sound like enum like constant so maybe Status {
val accepted = 1
}